docs/error-reporting/workflow-integrations/issue-tracking/jira.md

```diff
@@ -2,84 +2,97 @@
 id: jira
 title: Jira Integration with Backtrace
 sidebar_label: Jira
-description: Integrate Backtrace with Jira.
+description: Connect errors from Backtrace with issues in Jira to easily manage and track bug fixes.
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This guide goes through the steps necessary to integrate Backtrace with Jira. Setting up integration with the Jira ticketing system requires a valid Jira REST API URL, a user and either a password for the user or an API token.
+Integrate your Backtrace and Atlassian Jira accounts so you can create and update issues in Jira for errors reported to Backtrace.
```

Comment:
```suggestion
Integrate your Backtrace and Atlassian Jira accounts so that you can create and update issues in Jira for errors reported to Backtrace.
```